compiler/scenario-service/client/src/DA/Daml/LF/PrettyScenario.hs

```diff
@@ -825,14 +825,39 @@ renderValue world name = \case
         renderField (Field label mbValue) =
             renderValue world (name ++ [TL.toStrict label]) (fromJust mbValue)
 
+templateConName :: Identifier -> Maybe (LF.Qualified LF.TypeConName)
```

Comment:
This never returns `Nothing`. Can we please drop the `Maybe`? It would be an internal error if `Identifier` does not have the format we're expecting.